responses/sl/HassGetState.yaml

```diff
@@ -14,6 +14,18 @@ responses:
           Ne, {{ state.state_with_unit }}
         {% endif %}
 
+      someone_yesno: |
+        {% if query.matched %}
+          Da
+        {% else %}
+          Ne,
+          {% if state.attributes.get("untranslated_state") == "not_home" %}
```

Comment:
There's no need for this `if` if there's no difference. In English there's `John is away` and `John is at work`

Reply:
I am aware that not all responses are correct. I'll have to invest additional time to extensively work on improving the extensive grammar of Slovenian.

Since there are almost no users for these intents except for me, I will have ample time to test it thoroughly.
If you think it's inadequate, that's fine with me too. I just wanted to hurry because intents and templates in yaml will take up too much of my time during the week.

Reply:
Whatever you think is OK. My point was that the `if` is redundant.